modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging-from-44_{context}"]
+= Updating cluster logging from 4.4
+
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Cluster Logging Operator and the Elasticsearch Operator.
+
+When your cluster logging version is 4.4, you must update cluster logging as follows due to changes in the way that OLM provides operator packages.
+
+Additionally, it is recommended that you update Cluster Logging Operator before you update Elasticsearch Operator to avoid
+the need to restart each Elasticsearch pod manually, one by one.
+
+Perform each update, version by version, in the following order:
+
+. Update Cluster Logging Operator to the latest version of 4.4, then update to 4.5.
+. Update Cluster Logging Operator to latest version of 4.5, then update to 4.6.
+. Update Elasticsearch Operator to the latest version of 4.4, then update to 4.5.
+. Update Elasticsearch Operator to latest version of 4.5, then update to 4.6
```

Comment:
That is not correct. You can't upgrade following the order above.
The order should be.
Upgrade Openshift Cluster to 4.5
Update Cluster Logging Operator to the latest version of 4.4, then update to 4.5.
Update Elasticsearch Operator to the latest version of 4.4, then update to 4.5.
Upgrade Openshift Cluster to 4.5.
Update Cluster Logging Operator to latest version of 4.5, then update to 4.6.
Update Elasticsearch Operator to latest version of 4.5, then update to 4.6

Reply:
Done, but in the second instance of OpenShift Cluster, I think you have a typo. I think it should say 4.6, so that's what I added to the docs. Thank you.